cli/src/commands/split.rs

```diff
@@ -236,13 +240,19 @@ pub(crate) fn cmd_split(
             commit_builder.generate_new_change_id();
         }
         let description = if !args.message_paragraphs.is_empty() {
-            let description = join_message_paragraphs(&args.message_paragraphs);
-            if !description.is_empty() {
+            let mut description = join_message_paragraphs(&args.message_paragraphs);
+            if !description.is_empty() || args.editor {
                 commit_builder.set_description(description);
-                add_trailers(ui, &tx, &commit_builder)?
-            } else {
-                description
+                description = add_trailers(ui, &tx, &commit_builder)?;
+                if args.editor {
```

Comment:
Shouldn't we open an editor for the second commit as well?

Reply:
To be honest my preference was not to, but on thinking about it that's very unlikely to be expected behaviour, so I'll change it.